autogpt_platform/backend/backend/blocks/airtable/_api_test.py

```diff
@@ -1,7 +1,7 @@
 from os import getenv
 from uuid import uuid4
 
-import pytest
+import pytest  # type: ignore
```

Comment:
?

Reply:
Have you tried fixing your IDE's configuration? For everyone else including the CI type-check this is unnecessary, and unnecessary `# type: ignore` statements are bad for maintainability and in most cases also for reliability.